### ISSUE TITLE
Get Metadata "childitems" not available

### ISSUE THREAD

**Issue by iLikeAzureSQL** [Mmodarre/AzureDataFactoryHOL#13]
Looking at https://github.com/Mmodarre/AzureDataFactoryHOL/blob/master/Lab-guide/05-Get_Metadata_activity_filter_activity_and_complex_expressions/README.md

https://github.com/Mmodarre/AzureDataFactoryHOL/blob/master/Lab-guide/05-Get_Metadata_activity_filter_activity_and_complex_expressions/README.md

Child Items is not in the list to select from!

![image](https://user-images.githubusercontent.com/16943754/82505867-33606400-9ab3-11ea-8b00-eb6e2fd3ee9c.png)

**Comment by Mmodarre** [comment]
Thanks for brining this up. I think ADF has gone through a release at //Build! so this needs to be updated.

**Comment by iLikeAzureSQL** [comment]
I agree, perhaps going through the rest of the lab I find more bugs that might be related to ADF releases.
